internal/locate/region_request.go

```diff
@@ -1453,6 +1628,8 @@ func IsFakeRegionError(err *errorpb.Error) bool {
 	return err != nil && err.GetEpochNotMatch() != nil && len(err.GetEpochNotMatch().CurrentRegions) == 0
 }
 
+const slowLogSendReqTime = 100 * time.Millisecond
```

Comment:
This slow interval could be regarded together with `tikv_client_read_timeout`. Besides, we need to test the log number generated when there is something wrong, to find out if there would be a log flooding risk.

Reply:
> This slow interval could be regarded together with tikv_client_read_timeout.

Done.

> Besides, we need to test the log number generated when there is something wrong, to find out if there would be a log flooding risk.

I test this in 1 tikv server down situation, and the result is already in PR description. Since this PR add a slow log time threshold, so the generated log must be less than before.